### PR TITLE
fix: apply svgProps to Svg component when target is custom

### PR DESCRIPTION
**Pull request by jeetiss**
small fix for `svgProps` and components

### CODE REVIEW

File: packages/svgo-jsx/svgo-jsx.js [hunk 1]
```diff
@@ -96,7 +96,7 @@ const convertAttributes = (node, svgProps) => {
       props.set(name, JSON.stringify(value));
     }
   }
-  if (node.name === "svg" && svgProps) {
+  if ((node.name === "svg" || node.name === "Svg") && svgProps) {
```
Comment (TrySound) [inline packages/svgo-jsx/svgo-jsx.js:99]: toLowerCase() to cover all possible cases at ones

Comment (jeetiss) [reply packages/svgo-jsx/svgo-jsx.js:99]: Maybe will be better to check the hierarchy not an element name in this case? The SVG component can be named in various ways (Root, SVGRroot, whatever), so we inject the props only for the top-level element.

Comment (TrySound) [reply packages/svgo-jsx/svgo-jsx.js:99]: Yep, I guess it's find to spread into an element in the root. Then need to pass parentNode and check it's type for "root".

Comment (jeetiss) [reply packages/svgo-jsx/svgo-jsx.js]: Done, I'm not sure about argument order or maybe you have other suggestions, welcome 🙌🏻

File: packages/svgo-jsx/svgo-jsx.js [hunk 2]
```diff
@@ -122,13 +122,18 @@ const convertAttributes = (node, svgProps) => {
   return result;
 };
 
-const convertXastToJsx = (node, svgProps, components) => {
+const convertXastToJsx = (node, parent, svgProps, components) => {
```
Comment (TrySound) [inline packages/svgo-jsx/svgo-jsx.js:125]: Call it parentNode please. I use it everywhere in svgo.

File: packages/svgo-jsx/svgo-jsx.js [hunk 3]
```diff
@@ -96,7 +96,7 @@ const convertAttributes = (node, svgProps) => {
       props.set(name, JSON.stringify(value));
     }
   }
-  if ((node.name === "svg" || node.name === "Svg") && svgProps) {
+  if (parent && parent.type === "root" && svgProps) {
```
Comment (TrySound) [inline packages/svgo-jsx/svgo-jsx.js:99]: It's never null. Not sure how to enforce it with types but there is no types here yet.

**Comment by TrySound** [comment]
Thanks!

**Pull request merged**